cloud/pkg/cloudstream/streamserver.go

```diff
@@ -106,7 +109,23 @@ func (s *StreamServer) getContainerLogs(r *restful.Request, w *restful.Response)
 		}
 	}()
 
-	sessionKey := strings.Split(r.Request.Host, ":")[0]
+	// extract pod namespace and pod name from request
+	meta := strings.Split(r.Request.URL.Path, "/")
+	namespaceName := meta[2]
+	podName := meta[3]
+
+	kubeClient := client.GetKubeClient()
+	if kubeClient == nil {
+		klog.Fatalf("cannot get kube client\n")
```

Comment:
need to use `fatal` and stop the process?

Reply:
He is using `Fatalf()` this will exit the process and print a stacktrace. I don't get your point

Reply:
When you can't get the Kube Client you have a pretty much broken system you don't wont to work with.
So in my opinion he correct behavior is to exit.